### PR TITLE
Some general formatting fixes (punctuation, backticks, etc.)

### PR DESCRIPTION
**Pull request by agriyakhetarpal**
## Description

Ancillary changes from #169 that are best added in another PR, i.e., here.

## Changes made

Project names and terminologies interlaced with backticks, correct capitalization, spacing, punctuation, etc.

## Additional context

N/A

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -99,6 +99,6 @@ the [`jupyterlite_contents`](#jupyterlite-content) and the[`jupyterlite_dir`](#j
 options described above.
 
 This is an advanced feature and users are responsible for providing sensible command line options.
-The standard precedence between `jupyter lite build` CLI options and other means of configuration apply.
+The standard precedence between `jupyter lite build` CLI options and other means of configuration applies.
```
Comment (agriyakhetarpal) [inline docs/configuration.md:102]: This is better read as "The standard precedence (between `jupyter lite build` CLI options and other means of configuration) applies"; therefore, "precedence" as a singular noun should be accompanied with a plural predicate.

Comment (agriyakhetarpal) [reply docs/configuration.md:102]: Ah, that makes sense, too!

Comment (agriyakhetarpal) [inline docs/configuration.md:102]:
```suggestion
The standard precedence rules between `jupyter lite build` CLI options and other means of configuration apply.
```

Comment (steppi) [reply docs/configuration.md]: I think either your first suggestion, or my second one is fine, so pick what you like best.

Comment (agriyakhetarpal) [reply docs/configuration.md]: Picked yours and committed it!

**Review by steppi — APPROVED** [review]
Looks good. Thanks for separating this out into a separate PR @agriyakhetarpal.

**Pull request merged**